test/acceptance/interfaces/tdd.js

```diff
@@ -38,4 +38,24 @@ suite('Array', function(){
       done();
     });
   });
+
+  suite('tear down should be executed', function () {
+    var shouldBeChangeInTearDown;
+
+    setup(function () {
+      shouldBeChangeInTearDown = 5;
+    });
+
+    test('spec2', function () {
+      this.skip();
+    });
+
+    teardown(function () {
+      shouldBeChangeInTearDown = 10;
+    });
+
+    suiteTeardown(function () {
+      shouldBeChangeInTearDown.should.eql(10);
+    });
+  });
 });
```

Comment:
More importantly, if this is an issue in Runner and fixed in Runner, then it shouldn't be in the TDD-specific tests, but rather in the non-interface-specific tests. (In fact, unless we've pretty severely screwed up the way the interfaces interact with the rest of the system, all we should need to test in the interface-specific tests is that all their versions of the methods work, I think...)